src/bootlint.js

```diff
@@ -162,6 +164,51 @@ var LocationIndex = _location.LocationIndex;
         return runs;
     }
 
+    /**
+     * This function retrieves the browser window object and sets it to a variable.
+     * @returns {window object}
+     */
+    function getBrowserWindowObject() {
+        var theWindow = null;
+        try {
+            /*eslint-disable no-undef, block-scoped-var */
+            theWindow = window;// jshint ignore:line
+            /*eslint-enable no-undef, block-scoped-var */
+        }
+        catch (e) {
+            // deliberately do nothing
+        }
+
+        return theWindow;
+    }
+
+    function getSemverFromURI(pathSegments) {
+        return pathSegments.map(function (segment) {
+            var match = segment.match(/^\d+\.\d+\.\d+$/);
+            return match ? match[0] : null;
+        }).filter(function (match) {
+            return match !== null;
+        });
+    }
+
+    function getSemverFromPlugin(PLUGINS, globaljQuery) {
```

Comment:
I'm not sure I understand how to do this in this file. When I attempt to how I would normally expect to it just keeps throwing:
```shell
42:8   error  PLUGINS is defined but never used               no-unused-vars
215:33  error  PLUGINS is already declared in the upper scope  no-shadow
938:56  error  PLUGINS is already declared in the upper scope  no-shadow
987:51  error  PLUGINS is already declared in the upper scope  no-shadow
```